certified-connectors/Converter/Readme.md

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
Readme file is not in the correct format. Please refer to this [link](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) for readme template.